### PR TITLE
Cleanup unit tests

### PR DESCRIPTION
**Pull request by Abhishek332**
### **Type of changes:** improvement

moving some common things in common mock to reduce redundant code.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/AddObservabilityPage/AddObservabilityPage.test.tsx [hunk 1]
```diff
@@ -46,7 +47,7 @@ const MOCK_DATA = [
     provider: 'user',
   },
 ];
-const mockPush = jest.fn();
+
 const mockGoBack = jest.fn();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddObservabilityPage/AddObservabilityPage.test.tsx:51]: Can we move this to util as well as we are using this in multiple places?

File: openmetadata-ui/src/main/resources/ui/src/test/unit/mocks/rests/applicationAPI.mock.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   Permissions,
+  ProviderType,
+  ScheduleTimeline,
   ScheduleType,
-} from '../../generated/entity/applications/marketplace/appMarketPlaceDefinition';
-import { SearchIndexMappingLanguage } from '../../generated/settings/settings';
+  SearchIndexMappingLanguage,
+} from '../../../../generated/entity/applications/app';
+import { Status } from '../../../../generated/entity/applications/appRunRecord';
+// import {  } from '../../../../generated/entity/applications/appRunRecord';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/test/unit/mocks/rests/applicationAPI.mock.ts:10]: Remove this

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

**Comment by Abhishek332** [comment]
As per the discussion we are not moving common functions mock in utils

**Pull request closed without merging**